README.md

```diff
@@ -755,6 +761,78 @@ version, err := servicesManager.GetVersion()
 serviceId, err := servicesManager.GetServiceId()
 ```
 
+#### Fetching Users Details
+```go
+params := services.NewUsersParams()
+params.UserDetails.Name = "myUserName"
+
+user, notExists, err := serviceManager.GetUser(params)
+
+```
+
+#### Creating and Updating a User
+```go
+params := services.NewUsersParams()
+params.UserDetails.Name = "myUserName"
+params.UserDetails.Email = "myUser@jfrog.com"
+params.UserDetails.Password = "Password1"
+params.UserDetails.Admin = false
+params.UserDetails.Realm= "internal"
+params.UserDetails.ProfileUpdatable = true
+params.UserDetails.DisableUIAccess = false
+params.UserDetails.InternalPasswordDisabled = false
+params.UserDetails.groups = [2]string{"GroupA", "GroupB"}
+// Set to true in order to override exist user with the same name
+params.ReplaceExistUsers = false
+err := serviceManager.CreateUser(params)
+
+params.UserDetails.groups = [3]string{"GroupA", "GroupB", "GroupC"}
+err := serviceManager.UpdateUser(params)
+
+```
+
+#### Deleting a User
+```go
+err := serviceManager.DeleteUser("myUserName")
+```
+
+#### Fetching Group Details
+```go
+params := services.NewGroupParams()
+params.GroupDetails.Name = "myGroupName"
+// Set this param to true in order to receive the user names which associated with this group
```

Comment:
// Set this param to true in order to receive the user names which associated with this group
-->
// Set this param to true to receive the user names associated with this group